homeassistant/components/tuya/vacuum.py

```diff
@@ -205,7 +205,7 @@ def set_fan_speed(self, fan_speed: str, **kwargs: Any) -> None:
         self._send_command([{"code": DPCode.SUCTION, "value": fan_speed}])
 
     def send_command(
-        self, command: str, params: list[str] | None = None, **kwargs: Any
+        self, command: str, params: dict | list | None = None, **kwargs: Any
```

Comment:
Based on the service definition, it can be any dict or any list.

https://github.com/home-assistant/core/blob/0df30782a623204be2941da74ddee5bb110dd03b/homeassistant/components/vacuum/__init__.py#L146-L153